src/app/shared/launch.service.ts

```diff
@@ -91,7 +91,9 @@ unzip temp.zip`;
    */
   getPlanemoLocalInitString(workflowPath: string, versionName: string, primaryDescriptorPath: string, testParameterPath: string) {
     return (
-      this.getSharedZipString(workflowPath, versionName) + `\nplanemo workflow_job_init ${primaryDescriptorPath} -o ${testParameterPath}`
+      this.getSharedZipString(workflowPath, versionName) +
+      '\n# Note that if a parameter file has been provided by the workflow author, this will overwrite it.' +
+      `\nplanemo workflow_job_init --force ${primaryDescriptorPath} -o ${testParameterPath}`
```

Comment:
This doesn't feel right. The IWC workflows I looked at all have a test parameter file, so this command will fail for  all of them. I'm wondering if we shouldn't just give it a hard-coded name, like Dockstore.yml.

There's something else I missed in the first review that it might be easier to discuss live as well. Perhaps after/during our deploy?

Reply:
The command shouldn't fail with the new `--force` flag. 
Can discuss

Reply:
> The command shouldn't fail with the new `--force` flag. Can discuss

I did overlook that, but still wondering why we're overwriting the test parameter file by default -- wouldn't the user want to use it? But yeah, let's discuss this later.

Reply:
Ok, I updated since comparing with the WDL version, I realized that the WDL is actually using a hardcoded parameter file name so matching that behaviour in Galaxy seems simpler anyway than making it dynamic and match the user provided filename.